### ISSUE TITLE
[Paladín Retribution] Arrepentimiento ID:20066

### ISSUE THREAD

**Issue by paellaencol** [RoDNet/Bugtracker-Infinium-4.3.4#2862]
Enlace: http://wow.freakz.ro/spells.php?spell=20066&name=repentance

Descripción: Pone al objetivo enemigo en estado de meditación, lo que lo incapacita durante un máximo de 1 min. El daño que inflijan otras fuentes distintas a Censura despertará al objetivo. Solo funciona contra demonios, dragonantes, gigantes, humanoides y no-muertos.

Problema: Este efecto enmarcado en la categoría de desorientación (creo que tiene esta traducción al castellano), debería de eliminarse al recibir cualquier tipo de daño (excepto el daño provenient de Censura).

¿Qué está ocurriendo? Que si el objetivo de "Arrepentimiento" tiene un escudo de absorción, mientras el daño inflingido siga siendo absorbido por el escudo, arrepentimiento no se va.
Esto obviamente no debería ser así, al primer tick de daño, sea absorbido o no, este efecto debería desaparecer. Actualmente sólo expira cuando desaparece por tiempo, o cuando el efecto de absorción se rompe.
¿Este bug que está provocando? Que algunos paladines utilicen esta habilidad como segundo stun, contra clases que poseen efectos de absorción.

P.D. Si se necesita vídeo aclarativo de esto, puedo añadirlo, pero por privado.

**Comment by Wiinter** [comment]
No se si será así pero como yo lo interpreto es, si te absorve la pompa tu no recibes daño,por tanto es la pompa quien lo recibe. Si hiciera lo que tu dices en la descripción pondría que se quitaría cuando le atacasen no cuando le hicieran daño.

**Comment by paellaencol** [comment]
Sí, puede dar pie a esa interpretación, pero no es así, además es fácilmente comprobable con vídeos.

**Comment by Necrofilo** [comment]
Es lo mismo que picar bandera de BG con cualquier absorción encima..

**Comment by efectokoleos** [comment]
esto pienso k es del escudo mas k esta spell
